bin/setup_daq

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash -e
 
 uname -a
-lsb_release -a
+
+if ! command -v lsb_release &> /dev/null
```

Comment:
minor fix... general style is to have 'then' on the same line:

if ! command ...  &> /dev/null; then